src/main/java/com/sprint/mission/discodeit/entity/Channel.java

```diff
@@ -31,6 +32,15 @@ public Channel(ChannelCreateDTO channelCreateDTO, ChannelType type){
         this.type = type;
     }
 
+    //Pv 채널 생성
+    public Channel(PrivateChannelCreateDTO channelCreateDTO, ChannelType type){
```

Comment:
channelCreateDTO 내 필드 중에서 name만 필요한거라 이것만 인자로 받는게 좋을 것 같아요.  
엔터티가 DTO를 아는 것도 변경 전파 관련해서 좋지 않기도 하구요.  

```suggestion
    public Channel(String channelName, ChannelType type){
```